murageden/we-connect-v2-react-front-end
# #158820482 Fix Bug on Deleting a Business

#### What does this PR do?
Fixes a bug that was making it impossible to delete a user-owned business
#### Description of Task to be completed?
Ensure the functionality for deleting a user is working well with no errors
#### How should this be manually tested?
* Setup the application as described on `README`
* Switch to branch `fix-bug-on-deleting-a-business-#15882048`
* Register an account and login, and register a business
* On the list of businesses, a user should be able to click on an icon to delete business and delete a business successfully
#### Any background context you want to provide?
Previously, a bug was making this impossible
#### What are the relevant pivotal tracker stories?
Pivotal Tracker: https://www.pivotaltracker.com/n/projects/2177751
Story ID: 158820482
#### Questions:
How can this be improved?
